internal/exec/util/unit.go

```diff
@@ -100,13 +107,22 @@ func (u Util) EnableRuntimeUnit(unit types.Unit, target string) error {
 		u.DestDir = "/"
 	}
 
+	nodePath, err := u.JoinPath(SystemdRuntimeUnitWantsPath(target), unit.Name)
+	if err != nil {
+		return err
+	}
+	targetPath, err2 := u.JoinPath("/", SystemdRuntimeUnitsPath(), unit.Name)
```

Comment:
Minor/optional: we're using `:=` here anyway, so might as well make it `err` here too, right?

Reply:
Got my variable shadowing rules mixed up. :+1: to this